ml_peg/analysis/utils/decorators.py

```diff
@@ -665,6 +673,35 @@ def build_table_wrapper(*args, **kwargs) -> dict[str, Any]:
                 tooltip_header=tooltip_header,
             )
 
+            with open(MODELS_ROOT / "models.yml", encoding="utf8") as model_file:
```

Comment:
Maybe this code block should go in models.py, alongside `load_models` etc?

That has bits such as

```
    # Load models from registry YAML: models.yml
    with open(MODELS_ROOT / "models.yml") as file:
        all_models = yaml.safe_load(file)

    for name, cfg in get_subset(all_models, models).items():
```
which we may be able to use here too ?